src/lib/context.ts

```diff
@@ -230,12 +242,27 @@ export class AppContext {
         return this.span?._spanContext?.toTraceId();
     }
 
+    // allow override extra logger data, after ctx already initialized (ex. to add traceId from ctx)
+    setLoggerExtra(extra: Dict) {
```

Comment:
With this approach we are losing parent extra without any way to extend it (since property itself if private).

Maybe it would be useful to have functions like `addLoggerExtra(key, value)` and `clearLoggerExtra()` instead of this setter?